modules/core/include/opencv2/core/cvdef.h

```diff
@@ -898,15 +898,15 @@ class float16_t
     #endif
     }
 
-    static float16_t fromBits(ushort b)
```

Comment:
These methods has been removed from this class.

Reply:
Removed.